core/templates/pages/exploration-player-page/learner-experience/continue-button.component.ts

```diff
@@ -13,20 +13,24 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for the Continue button in exploration player and
+ * @fileoverview Component for the Continue button in exploration player and
  * editor.
  */
 
-angular.module('oppia').directive('continueButton', [
-  function() {
-    return {
-      restrict: 'E',
-      scope: {
-        onClickContinueButton: '&',
-        isLearnAgainButton: '&',
-        focusLabel: '@'
-      },
-      template: require('./continue-button.directive.html')
-    };
-  }
-]);
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+@Component({
+  selector: 'continue-button',
+  templateUrl: './continue-button.component.html',
+  styleUrls: []
+})
+export class ContinueButtonComponent {
+  @Input() focusLabel: string;
+  @Input() isLearnAgainButton: boolean = false;
+  @Output() clickContinueButton: EventEmitter<void> = new EventEmitter();
```

Comment:
rename this to continueButtonClickEventEmitter or notifyContineButtonCLicked or something around that.

Reply:
I think `continueButtonClick` should be sufficient here. The reason being we don't prefix the @Outputs with `on`. But when reading it we can just assume on was present and read it as `onContinueButtonClick`. https://angular.io/guide/styleguide#dont-prefix-output-properties.